fusee/fusee-secondary/src/main.c

```diff
@@ -77,7 +78,7 @@ int main(int argc, void **argv) {
 
     g_do_nxboot = loader_ctx->chainload_entrypoint == 0;
     if (g_do_nxboot) {
-        nxboot_main();
+        nxboot_main((uint8_t*)g_framebuffer);
```

Comment:
g_framebuffer is not defined in this file. This will cause a build error.
Do we need to just add `static void *g_framebuffer = (void *)0xC0000000;` to the file? or should the display also get initialized with it in a similar manner to fusee-primary?

Reply:
When I made this PR, `g_framebuffer` was defined there. However, it was moved between then and now, and I did an update to that fork which broke this (probably a bad idea). Tux has mentioned he'll merge and make any necessary changes, however, and the team is aware of this issue.

Reply:
Ah makes sense. The reason I asked is because I sent a PR to your fork adding PNG support and reducing the Fusee stage 2 payload size to ~470kb.

I was hoping somebody with hardware on hand can test it. I tested as much as I could of it on my host machine, but there's always the possibility that something might not work on real hardware.

Reply:
Right now, I think we're undecided on if we want to support multiple file formats, so I just stuck to BMP for now. I think support for other file formats has been brought up before, but we've just said to stick with BMP for the time being. I agree it might be a good idea to help shrink down stage2's size, though. Maybe get in touch with team and see what they feel about other image formats :)

Reply:
Cool. Yeah my main concern was the boot time. Any millisecond that can be shaved from the boot process will be very valuable.

The PR is available for grabs if the team decided to give it a shot.
I know they're extremely busy now with much more important stuff, but I thought I'd help with what I can :)